src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+  background: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__picture {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 0;
+  }
+
+  &__title {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-text-dark;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    font-style: normal;
+    font-weight: 400;
```

Comment:
Remove redundant default value. Fix all cases if present

Reply:
not fixed
<img width="673" alt="image" src="https://user-images.githubusercontent.com/53611910/236623234-0f6bef27-0c56-4101-9c9e-f15f53c450fa.png">